src/erlcloud_ddb2.erl

```diff
@@ -766,6 +777,29 @@ undynamize_expression(Expression, _) ->
 undynamize_expression_attribute_names(Names, _) ->
     Names.
 
+-spec undynamize_global_table_status(binary(), undynamize_opts()) -> table_status().
+undynamize_global_table_status(<<"CREATING">>, _) -> creating;
+undynamize_global_table_status(<<"UPDATING">>, _) -> updating;
+undynamize_global_table_status(<<"DELETING">>, _) -> deleting;
+undynamize_global_table_status(<<"ACTIVE">>, _)   -> active.
+
+-spec undynamize_replica_description(jsx:json_term(), undynamize_opts()) -> replica_description().
+undynamize_replica_description(ReplicaDescription, _) ->
+    #ddb2_replica_description{region_name = proplists:get_value(<<"RegionName">>, ReplicaDescription)}.
+
+-spec undynamize_replica_descriptions(jsx:json_term(), undynamize_opts()) -> [replica_description()].
+undynamize_replica_descriptions(ReplicaDescriptions, Opts) ->
+    [undynamize_replica_description(ReplicaDescription, Opts)
+     || ReplicaDescription <- ReplicaDescriptions].
+
+-spec undynamize_replica(jsx:json_term(), undynamize_opts()) -> replica().
+undynamize_replica(Replica, _) ->
+    #ddb2_replica{region_name = proplists:get_value(<<"RegionName">>, Replica)}.
+
+-spec undynamize_replicas(jsx:json_term(), undynamize_opts()) -> [replica_description()].
```

Comment:
Assuming `undynamize_replica/2` is -spec().ed OK, this function actually receives `([jsx:json_term()], _)` and outputs `[replica()]`.